include/openenclave/bits/defs.h

```diff
@@ -13,6 +13,13 @@
 #define OE_API_VERSION 2
 #endif
 
+/* OE_NO_RETURN */
+#if defined(__GNUC__)
```

Comment:
Is this specifically for the GNUC? What about clang?

Reply:
> "`__GNUC__` doesn't mean GCC specifically. All compilers that support GNU C extensions define it, including clang and ICC."

Reply:
Got it. Raising this just because I've seen some OE files use both `__GNUC__` and `__clang__`.

Reply:
Ya I think its just redundant